host/3.0/core-tools-publish.yml

```diff
@@ -374,7 +416,41 @@ stages:
         displayName: push dotnet images
         continueOnError: false
         condition: ne(variables['SkipProduction'], 'true')
-        
+
+      - bash: |
+          set -e
+          docker pull $SOURCE_REGISTRY/dotnet-isolated:$(PrivateVersion)-dotnet-isolated5.0-core-tools
+
+          docker tag $SOURCE_REGISTRY/dotnet-isolated:$(PrivateVersion)-dotnet-isolated5.0-core-tools $TARGET_REGISTRY/dotnet:$(TargetVersion)-dotnet-isolated5.0-core-tools
+          docker tag $SOURCE_REGISTRY/dotnet-isolated:$(PrivateVersion)-dotnet-isolated5.0-core-tools $TARGET_REGISTRY/dotnet:$(MajorVersion)-dotnet-isolated5.0-core-tools  
```

Comment:
Should this and above be "docker tag $SOURCE_REGISTRY/dotnet-isolated:$(PrivateVersion)-dotnet-isolated5.0-core-tools $TARGET_REGISTRY/**dotnet-isolated**:$(MajorVersion)-dotnet-isolated5.0-core-tools"?

Reply:
Thanks! good catch!